src/test/java/org/cactoos/iterator/NoNullsTest.java

```diff
@@ -23,72 +23,53 @@
  */
```

Comment:
@vzurauskas minor concern on commit message: right now the `@Rule` gets interpreted by GitHub, pointing to a user who has nothing to do with the project. Could you please amend the message and force-push?

Reply:
@scristalli Good catch. FIxed.